help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -573,7 +579,7 @@ for easy access click an item in this list:</p>
   start with the connection prefix, "C". This assigns the System name to an
   established CMRInet connection.</p>
 
-  <p>The next character in the System Name is the type of physical entity the
+  The next character in the System Name is the type of physical entity the
```

Comment:
Paragraphs should start with paragraph markers. Please restore it.

Reply:
Corrected lines in error.